src/data-scraper/index.ts

```diff
@@ -13,12 +13,23 @@ const httpsAgent = new HttpsAgent({
   rejectUnauthorized: false,
 });
 
-function getSysdigUrl(): string {
+// Soon to be deprecated
+function getSysdigV1Url(): string {
```

Comment:
nit: separating v1 into it's own file would make this easier to read, and make v1 easier to remove later.

Reply:
sure, will do that in a new PR.